bulk/eng_bulk_load-AssetRecord.csv

```diff
@@ -4,116 +4,122 @@ CE02SHBP-MJ01C-05-IP1,,Sensor,0,JBox Port J05 (LJ01D),JBox Port J05 (LJ01D),,,,,
+CGCON-BDCL12-15004,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2015-0004,,,,CE07SHSM-00004-DCL12,Oregon State University
+CGCON-BDCL12-15005,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2015-0005,,,,CE02SHSM-00004-DCL12,Oregon State University
+CGCON-BDCL12-15009,OL000304,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,9,,,,CE02SHSM-00001-DCL12 added 15000 to distinguish between Pioneer and Endurance,Oregon State University
+CGCON-BDCL12-17009,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2017-009,,,,TO BE DELETED,
+CGCON-BDCL12-50001,OL000282 R00052,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2015-0001,,,,CP04OSSM-00004-DCL12,Woods Hole Oceanographic Institution
+CGCON-BDCL12-50002,OL000100,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2015-0002,,,,,Woods Hole Oceanographic Institution
+CGCON-BDCL12-50003,OL000078,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2015-0003,,,,,Woods Hole Oceanographic Institution
+CGCON-BDCL12-50004,OL000264,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2015-0004,,,,CP03ISSM-00004-DCL12,Woods Hole Oceanographic Institution
+CGCON-BDCL12-50005,OL000232,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2015-0005,,,,CP01CNSM-00005-DCL12,Woods Hole Oceanographic Institution
+CGCON-BDCL12-50006,OL000525,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2015-0006,,,,,Woods Hole Oceanographic Institution
+CGCON-BDCL12-50007,OL000325,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2015-0007,,,,CE02SHSM-00003-DCL12,Oregon State University
+CGCON-BDCL12-50008,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2015-0008,,,,CE09OSSM-00005-SBD12,Oregon State University
+CGCON-BDCL12-50009,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2015-0009,,,,,Woods Hole Oceanographic Institution
+CGCON-BDCL12-60002,OL000595,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2016-0002,,,,,Woods Hole Oceanographic Institution
+CGCON-BDCL12-60701,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,60701,,,,DELETE?,
+CGCON-BDCL12-70001,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2017-0001,,,,CP04OSSM-00006-DCL12,Woods Hole Oceanographic Institution
+CGCON-BDCL12-70009,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2017-009,,,,,Woods Hole Oceanographic Institution
+CGCON-BDCL12-70009,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,UNKNOWN,,,,,Woods Hole Oceanographic Institution
```

Comment:
I believe the ASSET_UID for this one is supposed to be CGCON-BDCL12-99999 (according to eng_bulk_load-AssetRecord_NEW_2019-05-17 spreadsheet.